_posts/2016-04-17-sorted-pagination-in-cassandra.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+name: Sorted-pagination-in-Cassandra
+title: 'Sorted pagination in Cassandra'
+date: 2016-04-17 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/ring.png
+tags:
+- distributed systems
+- cassandra
+- data modeling
+
+---
+
+Cassandra is a fantastic database for different use cases. There are different situations when you need to twist Cassandra a little and studying one of those could be a helpful exercise to understand better what is Cassandra about. Databases are complex beasts, approaching them with the right level of abstraction is vital. Their final goal is not storing data per se, but make that data accessible. Those read patterns will define which database is the best tool for the job.
```

Comment:
'...and studying one of those could be a helpful exercise to understand better what is Cassandra about.'
->
'...and studying one of those could be a helpful exercise to better understand what Cassandra is about.'